app/controllers/mentee_profiles_controller.rb

```diff
@@ -1,6 +1,24 @@
 class MenteeProfilesController < ApplicationController
-	def show
-	end
-	def dashboard
-	end
+  def show
+    @user = User.find_by(id: params[:id])
+    @mentor = User.find_by(id: @user.matched_id)
+
+  end
+  def dashboard
+    @mentor = User.find_by(id: current_user.matched_id)
```

Comment:
same here you can use `User.find(current_user.matched_id))` . Also please add space between methods